tests/Pest.php

```diff
@@ -52,10 +52,9 @@
     }
 });
 
-uses(TestCase::class, RefreshDatabase::class)
-    ->beforeEach(fn () => DB::connection(DB::getDefaultConnection())
-        ->setQueryGrammar(
-            new PatchedSQLiteGrammar()
-        )
-    )
+uses(TestCase::class)
```

Comment:
The `RefreshDatabase` here was causing 2 migration commands to run. 

The first was TestBenches with the custom migration path (verbs and the example tables) and the second was with `RefreshDatabase` with the Laravel migration path (no tables). 

`RefreshDatabase` does not cause a problem with the in-memory database assumes a fresh database each run and just runs `migrate` which is kind of noop operation in TestBench.

However with a persistent storage like MySQL / Postgres, `RefreshDatabase` runs a `migrate:fresh` command, which wipes the verbs and example tables. 

There's no way to dynamically set the migration paths from `RefreshDatabase` afaik so we're better off removing it. We're running migrations [on every test](https://github.com/orchestral/testbench/issues/243#issuecomment-500177294) anyway so it's not needed.